src/uct/ib/rdmacm/rdmacm_cm.c

```diff
@@ -825,6 +825,10 @@ static uct_iface_ops_t uct_rdmacm_cm_iface_ops = {
     .iface_is_reachable       = (uct_iface_is_reachable_func_t)ucs_empty_function_return_zero
 };
 
+static uct_iface_internal_ops_t uct_rdmacm_cm_iface_internal_ops = {
```

Comment:
pls set default values for all fields, as in uct_base_iface_internal_ops

Reply:
@yosefe Done